package.json

```diff
@@ -45,6 +45,7 @@
     "ngx-countdown": "^11.0.3",
     "ngx-toastr": "^13.2.0",
     "rxjs": "~6.6.0",
+    "sweetalert2": "^11.0.9",
```

Comment:
1. Sweetalert is used for delete confirmation popup while you delete any kind of data it will asked you "are you sure want to delete".
2. This not toster Message this is for confirmation asking.